azure-iot-device/azure/iot/device/provisioning/pipeline/provisioning_pipeline.py

```diff
@@ -25,6 +25,7 @@ def __init__(self, security_client):
         Constructor for instantiating a pipeline
         :param security_client: The security client which stores credentials
         """
+        self._security_client = security_client
```

Comment:
Why are you saving this?  I don't see you using this anywhere.

Reply:
2 reasons :-
First I thought I missed this line (the other pipeline of iothub uses it)
Second I wanted to assert the instance of the security client

---
In reply to: [294440912](https://github.com/Azure/azure-iot-sdk-python-preview/pull/109#discussion_r294440912) [](ancestors = 294440912)